packages/app/components/feed/feed.md.tsx

```diff
@@ -30,7 +30,7 @@ import { Hidden } from "design-system/hidden";
 import { Tabs } from "design-system/tabs";
 import { CARD_DARK_SHADOW } from "design-system/theme";
 
-const CARD_HEIGHT = 848;
+const CARD_HEIGHT = 825;
```

Comment:
Any reason to change this? We sometimes needed to change these values but they are causing UI bugs. I think we should use normal the web approach instead rendering the lists with FlatList or VirtualizedList etc. I'll open a discussion on this. Just wanted to ping you if this won't break something.